tools/replay/android_main.cpp

```diff
@@ -84,6 +84,12 @@ void android_main(struct android_app* app)
 
     if (run)
     {
+        // Reinitialize logging with values retrieved from command line arguments
+        gfxrecon::util::Log::Settings log_settings;
+        GetLogSettings(arg_parser, log_settings);
+        gfxrecon::util::Log::Release();
+        gfxrecon::util::Log::Init(log_settings);
+
```

Comment:
It feels a little clunky that the standard codepath on Android even without this feature enabled will always be to init/release/init:
```cpp
gfxrecon::util::Log::Init();
//...
gfxrecon::util::Log::Release();
gfxrecon::util::Log::Init(log_settings);
```

Reply:
I have followed the exact same path as for desktop (I just copied these lines from `desktop_main.cpp` and pasted them at the equivalent same place)

Reply:
You make a convincing argument!